addons/account/models/account_move.py

```diff
@@ -2722,6 +2718,14 @@ def _stolen_move(self, vals):
             if command[0] == Command.SET:
                 yield from self.env['account.move.line'].browse(command[2]).move_id.ids
 
+    def _get_protected_vals(self, vals, records):
+        protected = set()
+        for fname in vals:
+            field = records._fields.get(fname)
+            if field.inverse or (field.compute and not field.readonly):
+                protected.update(self.pool.field_computed.get(field, [field]))
+        return [(protected, rec) for rec in records]
```

Comment:
![image](https://github.com/odoo/odoo/assets/21679159/0c000a53-7f67-4791-a1c7-35adb428202a)

Reply:
Tried removing it, but then I have things like this:
https://runbot.odoo.com/runbot/build/62215925

In this case, since the journal is being synced, the currency of the payment is recomputed.

Note that it looks like the populate seems to fail for the same kind of reason.